docs/pages/reference/core/createFrames.mdx

```diff
@@ -49,6 +49,17 @@ An array of middleware functions that are called before the frame handler and al
 
 Each middleware should return a promise that resolves to the next middleware, or a [Web API `Response`](https://developer.mozilla.org/en-US/docs/Web/API/Response), or a `FrameDefinition`.
 
+### `stateSigningSecret`
+
+- Type: `string`
+
+A secret key used to sign and verify the state in the frame message. If provided, state is automatically signed with this key and verified on each frame.
+
+- If the signature is present in state data:
+  - If the secret is not provided, the state will not be verified and used as is.
+  - If the secret is provided and signature does not match, the state will be set to `initialState`.
```

Comment:
I don't think this is expected behaviour - what do you think of throwing an error instead?

Or perhaps passing whether the signature is valid in a `isValid` property?

Reply:
I think throwing and error is cleaner since that should be treated as unauthorized access. Ok I will change that.